README.md

```diff
@@ -24,52 +24,181 @@ By default converting an image will output in 256 color, as 80 columns, and ASCI
 ```bash
 $ climage image.png
 ```
-![demo](https://raw.github.com/pnappa/CLImage/master/extra/warhol256ascii.png)
+![demo](extra/warhol256ascii.png)
 
 
 A nicer image can be obtained when enabling unicode and truecolor flags.
 ```bash
 $ climage --unicode --truecolour image.png
 ```
-![demo](https://raw.github.com/pnappa/CLImage/master/extra/warholtruecolorunicode.png)
+![demo](extra/warholtruecolorunicode.png)
 
 For display in TTYs such as the Linux terminal before starting X11 (also accessible by Ctrl-Alt-F3, etc), you should restrict yourself to 8 color, and ASCII only. As 8 and 16 colors are solely constructed out of system colors (often set by the terminal theme), it is recommended to select a palette for these, depending on what you have selected.
 ```bash
 $ climage --8color warhol.png --palette solarized
 ```
-![demo](https://raw.github.com/pnappa/CLImage/master/extra/warhol8colsolarized.png)
+![demo](extra/warhol8colsolarized.png)
 
 Further options may be found by running `climage --help`
 
 ## Python Library
 
-It is recommended that you use the Python library if converting a large number of images. Usage of the library mirrors that of the CLI tool.
+It is recommended that you use the Python library if converting a large number of images. Simple usage:
 
 ```python3
 import climage
 
-output = climage.convert('image.png', is_unicode=True)
+# Convert an image to a 50 character wide image.
+output = climage.convert('image.png', is_unicode=True, width=50)
```

Comment:
- [ ] Add this information to the setup.py script, so that when I publish to pip next, it'll have better info.